packages/social-trader/src/utils/helpers.ts

```diff
@@ -179,6 +179,11 @@ export const getRandomWords = (length: number = getRandomInteger(3, 8)) =>
 export const getRandomEmail = () =>
   `${getRandomWord()}@${getRandomWord()}.${getRandomWord()}`;
 
+export const partiallyHideEmail = (email: string): string => {
+  const [first, second] = email.split("@");
+  return `${first.slice(0, 3)}${"*".repeat(3)}@${second}`;
```

Comment:
partiallyHideEmail("bob@mail.ru") -> bob***@mail.ru

How long will I be test a framework for you?)))
Please, write tests 🙏🙏🙏